app/src/views/SurgeCatalogueOtherUAV/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { useMemo } from 'react';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+import { resolveToComponent } from '@ifrc-go/ui/utils';
+
+import SurgeCatalogueContainer from '#components/domain/SurgeCatalogueContainer';
+import SurgeContentContainer from '#components/domain/SurgeContentContainer';
+import Link from '#components/Link';
+
+import i18n from './i18n.json';
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const strings = useTranslation(i18n);
+
+    const imageList = useMemo(
+        () => ([
+            {
+                src: 'https://prddsgofilestorage.blob.core.windows.net/api/documents/surge/uav1.jpg',
+                caption: strings.uavImageOne,
+            },
+            {
+                src: 'https://prddsgofilestorage.blob.core.windows.net/api/documents/surge/uav2.jpg',
+                caption: strings.uavImageTwo,
+            },
+            {
+                src: 'https://prddsgofilestorage.blob.core.windows.net/api/documents/surge/uav3.jpg',
+                caption: strings.uavImageThree,
+            },
+            {
+                src: 'https://prddsgofilestorage.blob.core.windows.net/api/documents/surge/uav4.jpg',
+                caption: strings.uavImageFour,
+            },
+        ]),
+        [
+            strings.uavImageOne,
+            strings.uavImageTwo,
+            strings.uavImageThree,
+            strings.uavImageFour,
```

Comment:
These strings are missing values in the `i18n.json`

Reply:
Thanks for the fix, @frozenhelium